GPU/Common/SplineCommon.cpp

```diff
@@ -926,16 +926,19 @@ void DrawEngineCommon::SubmitSpline(const void *control_points, const void *indi
 	patch.patchFacing = patchFacing;
 
 	if (g_Config.bHardwareTessellation && g_Config.bHardwareTransform && !g_Config.bSoftwareRendering) {
-		float *pos = (float*)(decoded + 65536 * 18); // Size 3 float
-		float *tex = pos + count_u * count_v * 3; // Size 3 float
-		float *col = tex + count_u * count_v * 3; // Size 4 float
+		int stride = g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11 ? 4 : 3;
```

Comment:
I've never understood the ternary parenthesis paranoia.  If it's a single expression, it should be obvious.  I feel like an argument for `((a || b) ? c : d)` is naturally an argument for `(a + (b * c))`.

I agree about the parameter, since the backend is deciding it seems logical to ask it - or otherwise not let it decide, and force 4 for everyone.

-[Unknown]